src/lib/filter/filter.ts

```diff
@@ -139,6 +140,12 @@ export function shouldSkipRequest(
         `but ignored namespaces include '${JSON.stringify(ignoredNamespaces)}'.`
       ) :
 
+    missingCarriableNamespace(capabilityNamespaces, obj!) ? 
```

Comment:
```suggestion
    missingCarriableNamespace(capabilityNamespaces, obj) ? 
```
Looks like an extra `!` snuck in..?

Reply:
This is going to be defined no matter what, but the type of the Kubernetes Object is 

Argument of type 'KubernetesObject | undefined' is not assignable to parameter of type 'KubernetesObject'.
  Type 'undefined' is not assignable to type 'KubernetesObject'.ts(2345)
const obj: KubernetesObject | undefined.

So we have to use the non-null here

Reply:
Oh, yeah.  Eew.  Then when we start specifying types in all the other adjudicators we're going to have to start putting these non-null assertions _everywhere_. :thinking: 

What would you think of us just putting the assertion up on line #57-ish so we don't have to have those bangs everywhere?  Would then cover all the other adjudicator calls too..?

Reply:
Yep, love the update.  `!`-pocolyse avoided!   :hand_over_mouth: :+1: